File: pano_progra/practica_1.py
valores = []
valores_nulos = []


def limite():
  valor_a_revisar = float('inf')
  while valor_a_revisar != 0:
    valor_a_revisar = float(input("Ingresa el valor a revisar (0 para terminar): "))
    if valor_a_revisar == 0:
      break
          
    if valor_a_revisar >= limite_superior or valor_a_revisar <= limite_inferior:
      valores_nulos.append(valor_a_revisar)
    else:
      valores.append(valor_a_revisar)

  print(f"\nLos valores fuera de los limites: {valores_nulos}")
  print(f"Los valores dentro de los limites: {valores}")
  print(f"La suma de los valores dentro de los limites: {sum(valores)}")

limite_superior = float(input("Ingresa el limite superior: "))
limite_inferior = float(input("Ingresa el limite inferior: "))


if limite_inferior < limite_superior:
  limite()

print("error")

File: pano_progra/practica_1.cs
using System;
using System.Collections.Generic;
using System.Linq;

class Program
{
    static void Main(string[] args)
    {
        List<double> valores = new List<double>();
        List<double> valoresNulos = new List<double>();

        Console.Write("Ingresa el limite superior: ");
        double limiteSuperior = Convert.ToDouble(Console.ReadLine());

        Console.Write("Ingresa el limite inferior: ");
        double limiteInferior = Convert.ToDouble(Console.ReadLine());

        if (limiteInferior < limiteSuperior)
        {
            double valorARevisar = double.PositiveInfinity;

            while (valorARevisar != 0)
            {
                Console.Write("Ingresa el valor a revisar (0 para terminar): ");
                valorARevisar = Convert.ToDouble(Console.ReadLine());

                if (valorARevisar == 0)
                    break;

                if (valorARevisar >= limiteSuperior || valorARevisar <= limiteInferior)
                {
                    valoresNulos.Add(valorARevisar);
                }
                else
                {
                    valores.Add(valorARevisar);
                }
            }

            Console.WriteLine("\nLos valores fuera de los limites: " + string.Join(", ", valoresNulos));
            Console.WriteLine("Los valores dentro de los limites: " + string.Join(", ", valores));
            Console.WriteLine("La suma de los valores dentro de los limites: " + valores.Sum());
        }
        else
        {
            Console.WriteLine("Error: El límite inferior debe ser menor que el límite superior.");
        }
    }
}
